devicon.json

```diff
@@ -11289,5 +11289,23 @@
                 "alias": "plain"
             }
         ]
+    },
+    {
+        "name": "stackblitz",
+        "altnames": [],
+        "tags": [
+            "Full-Stack-Development",
+            "Online-IDE"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
```

Comment:
If you want to use original as a font, you need to make sure it is monocolored, and has a single unified `<path>` element, as outlined in our [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions) wiki post